server/conf/application.conf

```diff
@@ -345,6 +345,7 @@ play.filters {
   enabled += play.filters.gzip.GzipFilter
   enabled += filters.UnsupportedBrowserFilter
   enabled += filters.SettingsFilter
+  enabled += filters.SessionIdFilter
```

Comment:
**issue (blocking):** please guard the new redirect behavior with a feature flag if we're enabling this filter immediately. We won't know for sure that it doesn't break integrations with the various identity providers until it gets exercised with them.